mender-convert.sh

```diff
@@ -332,6 +332,11 @@ do_make_sdimg_raspberrypi3() {
   sudo install -d -m 755 ${sdimg_primary_dir}/data
 
   set_fstab $device_type
+
+  [[ $keep -eq "-k" ]] && { rm -rf $output_dir/boot.vfat\
+     $output_dir/cmdline.txt  $output_dir/config.txt\
```

Comment:
In this place there are no cmdline.txt & config.txt files. You deal with them in stage 5 for RP PI3.

Reply:
config.txt and cmdline.txt are also copied to output directory in stage 2 for RPi3.

Reply:
Right!